docs/plutus-report/plutus.tex

```diff
@@ -66,7 +66,10 @@
 \title{The Plutus Platform \\
   {\large \sc An IOHK technical report}}
 \date{}
-\author{}
+\author{
+  Michael Peyton Jones \\ {\small \texttt{michael.peyton-jones@iohk.io}} \\
+  \and Roman Kireev \\ {\small \texttt{roman.kireev@iohk.io}}
```

Comment:
I don't think my tiny indirect contribution should lead to my name being on the face of this document. You could put me in an "Acknowledgements" section, if you wish, but that is also not necessary.

Reply:
I'm on the fence about having authors at all, but insofar as it has authors you've contributed the most apart from me so you get to be in it :P

Reply:
So unfair.